packages/services/api/src/modules/lab/module.graphql.ts

```diff
@@ -8,4 +8,45 @@ export default gql`
     schema: String!
     mocks: JSON
   }
+
+  type PreflightScript {
+    id: ID!
+    sourceCode: String!
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    # TODO: should I add this field?
+    # createdBy: User!
```

Comment:
TODO @dimaMachina remove this. We don't need it since right now you can only have one preflight script per organization. We might reconsider later on.

Reply:
removed